code/modules/shuttle/shuttle.dm

```diff
@@ -283,7 +283,7 @@
 	var/launch_status = NOLAUNCH
 
 	///Whether or not you want your ship to knock people down, and also whether it will throw them several tiles upon launching.
-	var/list/movement_force = list("KNOCKDOWN" = 3, "THROW" = 0)
+	var/list/movement_force = list("KNOCKDOWN" = 3, "THROW" = 14)
```

Comment:
14? Get out.

Reply:
Weee being thrown at hyper speed

Reply:
The shuttle literally takes off at light speed. If I wanted to be realistic about it, you would immediately be gibbed by the velocity and mass of the shuttle colliding with you. This works.